stories/text/text.stories.js

```diff
@@ -0,0 +1,172 @@
+    quisque id diam vel quam elementum pulvinar. Eget aliquet nibh praesent tristique magna. Id interdum velit laoreet
+    id donec ultrices tincidunt arcu. Suspendisse ultrices gravida dictum fusce ut placerat orci nulla. Montes nascetur
+    ridiculus mus mauris vitae ultricies. In nisl nisi scelerisque eu. Suspendisse interdum consectetur libero id
+    faucibus. Ut lectus arcu bibendum at varius. Placerat vestibulum lectus mauris ultrices eros in cursus turpis massa.
+    Dolor magna eget est lorem ipsum dolor sit. Aliquam nulla facilisi cras fermentum odio eu feugiat pretium nibh.
+    Augue neque gravida in fermentum et. Proin fermentum leo vel orci porta non pulvinar.</p>
+
+    <h3>3 max lines</h3>
+    <p class="fd-text" style="display: -webkit-box; -webkit-box-orient: vertical; -webkit-line-clamp: 3; overflow: hidden;">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+    eiusmod tempor incididunt ut labore et dolore magna aliqua. Malesuada bibendum arcu vitae elementum. Ut etiam sit
+    amet nisl. Laoreet suspendisse interdum consectetur libero id faucibus nisl. Tellus in metus vulputate eu
+    scelerisque felis imperdiet proin fermentum. Ullamcorper morbi tincidunt ornare massa eget egestas purus viverra
+    accumsan. Mauris vitae ultricies leo integer malesuada nunc vel risus. Nisi est sit amet facilisis. Eu turpis
+    egestas pretium aenean. Nunc id cursus metus aliquam eleifend. Placerat in egestas erat imperdiet. Etiam dignissim
+    diam quis enim lobortis scelerisque. A erat nam at lectus. Amet nisl purus in mollis nunc sed id. Purus semper eget
+    duis at tellus at urna.
+    Aliquet eget sit amet tellus cras adipiscing enim. Risus ultricies tristique nulla aliquet enim tortor. Varius quam
+    quisque id diam vel quam elementum pulvinar. Eget aliquet nibh praesent tristique magna. Id interdum velit laoreet
+    id donec ultrices tincidunt arcu. Suspendisse ultrices gravida dictum fusce ut placerat orci nulla. Montes nascetur
+    ridiculus mus mauris vitae ultricies. In nisl nisi scelerisque eu. Suspendisse interdum consectetur libero id
+    faucibus. Ut lectus arcu bibendum at varius. Placerat vestibulum lectus mauris ultrices eros in cursus turpis massa.
+    Dolor magna eget est lorem ipsum dolor sit. Aliquam nulla facilisi cras fermentum odio eu feugiat pretium nibh.
+    Augue neque gravida in fermentum et. Proin fermentum leo vel orci porta non pulvinar.</p>
+`;
+
+maxLines.storyName = 'Max lines';
+maxLines.parameters = {
+    docs: {
+        iframeHeight: 300,
+        storyDescription: `You can define maximum number of lines for text, after reaching the limit text will be
+        truncated and ellipsis will be added. To set maximum numbers you should use these styles:
+        <code class="docs-code">
+        display: -webkit-box; -webkit-box-orient: vertical; -webkit-line-clamp: [number of lines]; overflow: hidden;
+        </code>`
```

Comment:
This should be in a `fd-text--modifier` class.

Reply:
@jbadan why? It's a common recipe for line clamp, it's not a unique mechanic for the text component.

Reply:
Then maybe this isn't a component, but a pattern?

Reply:
Hey, @jbadan, 
I think maybe there is a misunderstanding about these examples and why we have so different thoughts about those styles?

Just in case, I'll try to clarify my reason to add them.
I had a task to create a Text component styles, in the documentation described only base 4 style rules and selected state.

Also, in fiori designs were sections about wrapping, maxlines, hyphenation, but this functionality is the default for any text in html+css, you can control it as easily as setting color or font-size, but because it was mentioned in the design doc I added these examples, to show how it could be achieved. I saw similar examples here as well, for example showing icon size change by changing font-size via inline styles.

The reason to not create specific classes for those cases, that's this is not a component functionality, those are things that could be done with this component, the same as color change, font-size change and etc.

Reply:
@EvilAlexei, I understand your point of view but I'd rather remove these examples and stick with the example from the specs than providing all these examples with inline styling. You only need the styling that comes from the reset function, the text selection rules and the truncation (word by word). If the user wants to modify something let them figure out how to do it. We only show them how to achieve Fiori 3 case using classes.